crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -136,6 +136,50 @@ impl Node {
         Ok(())
     }
 
+    pub fn propose_dlc_channel_collaborative_settlement(
```

Comment:
nit: I'd prefer an aligned naming here. either use `collaborative_settlement` or `close_offer`

--
`initiate_propose_dlc_channel_close_offer` and `initiate_accept_dlc_channel_close_offer`
or
`propose_dlc_channel_collaborative_settlement` and `initiate_accept_dlc_channel_collaborative_settlement`